pallets/portfolio/src/lib.rs

```diff
@@ -239,8 +253,12 @@ decl_module! {
             to: PortfolioId,
             items: Vec<MovePortfolioItem<<T as CommonTrait>::Balance>>,
         ) -> DispatchResult {
-            let sender = ensure_signed(origin)?;
-            let did = Context::current_identity_or::<Identity<T>>(&sender)?;
+            let PermissionedCallOriginData {
+                primary_did,
+                secondary_key,
+                ..
+            } = Identity::<T>::ensure_origin_call_permissions(origin)?;
```

Comment:
Similar - should we check the secondary_key (if caller is a sk) has permission to the from and to portfolios?

Reply:
This is done by `ensure_portfolio_validity` a few lines below.